superset-frontend/src/components/ErrorMessageWithStackTrace.tsx

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
this file is moved from the old jsx file, major changes are making it a functional component and replacing the state object with a useState hook. Note that the old prop `showStackTrace` was unused except to initialize the state, so i removed it